components/script/dom/audioparam.rs

```diff
@@ -106,6 +112,22 @@ impl AudioParamMethods for AudioParam {
 
     // https://webaudio.github.io/web-audio-api/#dom-audioparam-automationrate
     fn SetAutomationRate(&self, automation_rate: AutomationRate) {
+        if automation_rate == AutomationRate::A_rate {
+            if self.node_type == AudioNodeType::AudioListenerNode {
+                // ignore
+                return;
```

Comment:
This is supposed to be ignored on the servo-media side, which already happens. We don't need this check.